Import/Esri/ArcGISRuntime/Toolkit/Controls/CppApi/TimeSlider.qml

```diff
@@ -0,0 +1,485 @@
+/*******************************************************************************
+ *  Copyright 2012-2018 Esri
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ ******************************************************************************/
+
+import QtQuick 2.6
+import QtQuick.Controls 2.2
+import QtQuick.Window 2.2
+import QtQuick.Layouts 1.1
+import Esri.ArcGISRuntime 100.3
+import Esri.ArcGISRuntime.Toolkit.CppApi 100.3
+
+/*!
+    \qmltype TimeSlider
+    \ingroup ArcGISQtToolkit
+    \ingroup ArcGISQtToolkitQmlApi
+    \inqmlmodule Esri.ArcGISRuntime.Toolkit.Controls
+    \since Esri.ArcGISRutime 100.3
+    \brief
+*/
+Item {
+    id: root
+    enabled: controller.startStep !== -1
+    clip: true
+    height: backgroundRectangle.height
+
+    property real scaleFactor: (Screen.logicalPixelDensity * 25.4) / (Qt.platform.os === "windows" ? 96 : 72)
+
+    property color textColor: "black"
+    property string fontFamily : "helvetica"
+    property int pixelSize : 12
```

Comment:
the problem is that the user will set this from their code... so if we apply any scaling here it will get lost if thee user sets their own value. Although obviously, if they set a value which has been scaled we will scale it twice. Should we expect scaled or not scaled do you think?

Reply:
That makes sense.  `pixelSizeInDips` would make sense for the name then.